src/libraries/System.Runtime/tests/System/Reflection/BindingFlagsDoNotWrap.cs

```diff
@@ -32,6 +39,14 @@ public static void ConstructorInvokeStringCtor()
             TestDoNotWrap<ArgumentNullException>((bf) => c.Invoke(bf, null, new object[] { null, 0, 0 }, null));
         }
 
+        [Fact]
+        public static void ConstructorInvokeStringCtorTwoArgs()
+        {
+            // Code coverage: Project N - String constructors go through a separate code path.
```

Comment:
Is this comment still relevant?

Reply:
Probably not. It was copied from the method above. I'll remove it.